src/dslite_parser.rs

```diff
@@ -131,8 +131,8 @@ pub struct Module {
 
 fn parse_cpu_instance(el: &Element, root_file: &Path) -> Option<Module> {
     assert_eq!(el.name, "instance");
-    let base = uw!(utils::parse_u32(uw!(el.attributes.get("baseaddr"))));
-    assert_eq!(base, 0);
+    let _base = uw!(utils::parse_u32(uw!(el.attributes.get("baseaddr"))));
+    //assert_eq!(base, 0);
```

Comment:
Are you sure that this is ok to ignore baseaddr? In the old files it was always 0, but what happens if it's not?

Reply:
In the old code baseaddr was only used for that assertion. I removed it because in the newly-added [device file](https://github.com/pftbest/msp430_svd/pull/8/commits/e090d5176e8a2fdf2edf56d5abf3ba0709af951e#diff-3191ca635a18a445cc4afb6dffc0b72c) the CPU instances all have non-zero base addresses. I'm not sure if other places in the code silently breaks because the assumption that all CPU base addresses are zero is broken, but I can investigate further if needed. Do you know of any such cases?

Reply:
On closer inspection the base address of the "CPU instance", or module, affects the offset address of the registers and should be reflected in the peripheral base addresses of the output SVD.

Reply:
That's what I feared. I was able to get away with a hack because old files didn't have base address, but now it seems we need to handle it properly.